common/test/metadata/MetaDataMatcher.scala

```diff
@@ -40,4 +40,26 @@ object MetaDataMatcher extends Matchers  {
 
   }
 
+  def ensureDeepLink(result: Future[Result], expectedLink: String) {
+    val body = Jsoup.parseBodyFragment(contentAsString(result))
+    status(result) should be(200)
+    val script = body.getElementsByAttributeValue("href", expectedLink)
+    script.size() should be(1)
+  }
+
+  def ensureNoDeepLink(result: Future[Result], nonExistantLink: String) {
+    val body = Jsoup.parseBodyFragment(contentAsString(result))
+    status(result) should be(200)
+    val script = body.getElementsByAttributeValue("href", nonExistantLink)
+    script.size() should be(0)
```

Comment:
testing for the absence of something very specific can be dangerous - if we want no deep link at all we should look for something starting with app:// etc?